Marlin/src/feature/tmc_util.cpp

```diff
@@ -968,7 +968,7 @@
     #endif
     TMC_REPORT("CS actual",          TMC_CS_ACTUAL);
     TMC_REPORT("PWM scale",          TMC_PWM_SCALE);
-    #if HAS_DRIVER(TMC2130) || HAS_DRIVER(TMC2224) || HAS_DRIVER(TMC2660) || HAS_TMC220x
```

Comment:
@thinkyhead
Could it be that **_TMC2224_** was confused here with **TMC2240**?
Because **_TMC2224_** has vsense according to the corresponding [datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/TMC2202_TMC2208_TMC2224_datasheet_rev1.14.pdf), whereas **TMC2240** only has _isense_ according to its [datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/tmc2240_datasheet.pdf).

Please don't quote me on that, though, as I'am not 100% sure. 😅